content/post/Android_Root101.en.md

```diff
@@ -1,31 +1,32 @@
 +++
 title = "A Guide to Flashing Custom ROMs [Android]"
-slug = "Flashing Custom ROMs 101"
+slug = "custom-rom-101"
 date = "2018-02-25T00:00:00"
+image = "img/rohanvtk/csrom.jpg"
 description = "Sick and tired of the bloatware on your current OEM ROM. Head over and learn how to flash custom ROMs."
```

Comment:
Somewhere I think this blog would be a little better for new people if we can have one line definitions for words and annotations used in the blog. Sure, people can Google that stuff but they would then have no reason to read a 101 guide.

Reply:
Sure! I'll work on it. I realize you're 💯 correct about this.

Reply:
@Sc0rpi0n101

Reply:
Yeah, that was also my point xD
along with some others :rofl: 
Waiting for an update then